### DISCUSSION TITLE
Migrating from WebStorm

### DISCUSSION THREAD

**Discussion by akoskm** [zed-industries/zed#21278] [Feature Requests]
Hi community 👋

I'm new to Zed and wondered if there's a plan to support WebStorm run configurations. We use a bunch of these to run a microservice architecture:

![image](https://github.com/user-attachments/assets/368df9d2-abea-4af4-b625-b9acb7e375ad)

Every service starts with its terminal, and you can start/debug/restart the processes separately.

I haven't found any extensions that list the services from the .idea configuration file, but it's a simple XML file, so the task doesn't seem impossible.

Any idea how to get this working or directions on making an extension for this?

Appreciate the answers!

**Comment by SomeoneToIgnore** [comment]
It's not a very elaborate pointer, but could be a start of something bigger: Zed tries to import VSCode tasks watching a specific file :

https://github.com/zed-industries/zed/blob/6cba467a4e218e85180ce271219d47b1923402de/crates/project/src/project_settings.rs#L442-L462

This is not a plugin though, but at least, if needed, could be extended to do similar things with the intellij files.

**Reply by akoskm** [reply]
Unfortunately, I don't know Rust, so this isn't the path for me right now, but someone else might find it useful. Thanks for sharing this, Kirill!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
